shared/cfg/base.cfg

```diff
@@ -300,6 +300,9 @@ vnc_autoport = yes
 
 # Guest VGA type (cirrus,  std, vmware, qxl, xenfb, none)
 vga = std
+# Whether use '-vga' expression to represent the VGA device,
+# if no, using '-device' expression.
+vga_use_legacy_expression = no
```

Comment:
I don't really remember when `-device` option for gpus was added, do you think it'd make sense to detect the `-device` presence and in case it's not found fallback to `-vga`?

I mean I'm fine with it as is as people using older systems can always set that, I'm just asking how likely people are to need this.

Reply:
I checked that the option has been supported in qemu 0.12. As the check is easy to be done, I think we could have it.

Reply:
0.12 is quite long time ago (which RHEL was it?) I'm not sure we need to check the version and users could set it manually....

Reply:
It would be RHEL 6, anyway, I have had the below check in the new version, thanks.
```python
# fallback if qemu not has such a device
elif not devices.has_device(vga_dev):
    fallback = True
```